posts/2023-05-24-api-automation-testing-tools-comparison.adoc

```diff
@@ -7,7 +7,7 @@ v1.0, 2023-06-05
 :tags: [api, test automation, testing techniques]
 
 
-Internal projects at Lunatech play a crucial role in fostering growth, innovation, and knowledge-sharing within employees. These projects serve as the sandbox where new joiners are given the opportunity to explore, learn, and contribute in a hands-on manner.
+Internal projects at Lunatech play a crucial role in fostering growth, innovation, and knowledge-sharing between employees. These projects serve as the sandbox where new joiners are given the opportunity to explore, learn, and contribute in a hands-on manner.
 ++++
 </br>
 ++++
```

Comment:
Yes, I use this to add a line spacing between the paragraphs. 
because with only a blank line it just adds the new paragraph, but not the line spacing